bbc/simorgh
# Add Focus&Hover state to logo on header

- add focus state (styling) to logo
- update snapshots and tests

* [x] Fixes https://github.com/bbc/simorgh/issues/344
* [x] Fixes https://github.com/bbc/simorgh/issues/343


**Test steps**
1. git checkout add-logo-focus-state-to-header
2. `npm run dev` 
3. Go to https://localhost:7080/article/scenario-27
4. Tab onto the header 
5. Observe the underline fucntionality matches that of the screenshot/UX designs.
6. Repeat steps 4&5 hoverer instead hover on the header instead of tabbing onto it.
